add statosio

a6b8 opened the pull request:
- fix tables
- add script tag with sri
- json new line

MattIPv4 commented:
FYI, in future you don't need to close the PR and open a new one, the old one was updated and was fine to leave open for a second review 👍

MattIPv4 commented on statosio/change-style-to-dark-mode/index.md:
```diff
@@ -0,0 +1,251 @@
+## Result
+
+![# d3.statosio](https://d3.statosio.com/assets/images/example-customize-400.jpg)<br>
+[Change style to dark-mode](https://d3.statosio.com/tutorials/change-style.html)
```
```suggestion
```
I don't think we need this link, as this is just the same tutorial as what we're currently showing?

MattIPv4 commented on statosio/create-a-simple-point-chart/index.md:
```diff
@@ -0,0 +1,188 @@
+## Result
+
+![# d3.statosio](https://d3.statosio.com/assets/images/example-point-400.jpg)<br>
+[Create a simple point chart](https://d3.statosio.com/tutorials/simple-point-chart.html)
```
```suggestion
```
No need to link to identical content, just confuses the reader

MattIPv4 commented on statosio/create-a-simple-stacked-bar-chart/index.md:
```diff
@@ -0,0 +1,183 @@
+## Result
+
+![# d3.statosio](https://d3.statosio.com/assets/images/example-stacked-bar-400.jpg)<br>
+[Create simple stacked bar chart](https://d3.statosio.com/tutorials/simple-stacked-bar-chart.html)
```
```suggestion
```
No need for link

MattIPv4 replied:
This link still needs removing

MattIPv4 commented on statosio/create-a-simple-stacked-bar-chart/index.md:
```diff
@@ -0,0 +1,183 @@
+|:---|:---|:---|
+| [showAverage](https://d3.statosio.com/options/show__average.html) | Calculate and show average line | false |
+
+List of all "show" Options: [here](https://d3.statosio.com/options/index.html#show)s
```
```suggestion
List of all "show" Options: [here](https://d3.statosio.com/options/index.html#show)
```
Random `s`?

MattIPv4 commented on statosio/select-and-change-position-of-columns/index.md:
```diff
@@ -0,0 +1,226 @@
+## Result
+
+![# d3.statosio](https://d3.statosio.com/assets/images/example-select-400.jpg)<br>
+[Select and change position of columns](https://d3.statosio.com/tutorials/select-data.html)
```
```suggestion
```
Remove duplicate content link

MattIPv4 commented on statosio/select-and-change-position-of-columns/index.md:
```diff
@@ -0,0 +1,226 @@
+| [showAverage](https://d3.statosio.com/options/show__average.html) | Calculate and show average line | false |
+
+List of all "show" Options: [here](https://d3.statosio.com/options/index.html#show)
+
+
```
```suggestion
```
Remove extra blank lines at end of file

MattIPv4 commented on statosio/sort-dataset-by-values/index.md:
```diff
@@ -0,0 +1,225 @@
+## Result
+
+![# d3.statosio](https://d3.statosio.com/assets/images/example-sort-400.jpg)<br>
+[Sort dataset by values](https://d3.statosio.com/tutorials/sort-data.html)
```
```suggestion
```
Rm dupe content link

MattIPv4 reviewed (changes requested):
In a few places you use bullet points when they aren't really part of a list -- this seems like an odd style choice for the content? If they're meant to be subheadings, maybe bold would be better?

E.g. `- Option explained`

a6b8 commented:
> In a few places you use bullet points when they aren't really part of a list -- this seems like an odd style choice for the content? If they're meant to be subheadings, maybe bold would be better?
> 
> E.g. `- Option explained`


thanks for the fast reply

- removed bullet points.
- removed links
- removed "s"

MattIPv4 commented on statosio/create-a-simple-bar-chart/index.md:
```diff
@@ -0,0 +1,182 @@
+
+## Result
+
+![# d3.statosio](https://d3.statosio.com/assets/images/example-bar-400.jpg)<br>
```
```suggestion
![# d3.statosio](https://d3.statosio.com/assets/images/example-bar-400.jpg)
```
br not needed

MattIPv4 commented on statosio/create-a-simple-bar-chart/index.md:
```````diff
@@ -0,0 +1,182 @@
+
+```javascript
+d3.statosio( source, x, y, options )
+``````
```````
Extra fences need removing

MattIPv4 commented on statosio/create-a-simple-point-chart/index.md:
```diff
@@ -0,0 +1,187 @@
+
+## Result
+
+![# d3.statosio](https://d3.statosio.com/assets/images/example-point-400.jpg)<br>
```
```suggestion
![# d3.statosio](https://d3.statosio.com/assets/images/example-point-400.jpg)
```
br not needed

MattIPv4 commented on statosio/create-a-simple-point-chart/index.md:
```````diff
@@ -0,0 +1,187 @@
+
+```javascript
+d3.statosio( source, x, y, options )
+``````
```````
Extra fences need removing

MattIPv4 commented on statosio/create-a-simple-stacked-bar-chart/index.md:
```diff
@@ -0,0 +1,183 @@
+
+## Result
+
+![# d3.statosio](https://d3.statosio.com/assets/images/example-stacked-bar-400.jpg)<br>
```
```suggestion
![# d3.statosio](https://d3.statosio.com/assets/images/example-stacked-bar-400.jpg)
```
br not needed

MattIPv4 commented on statosio/create-a-simple-stacked-bar-chart/index.md:
```````diff
@@ -0,0 +1,183 @@
+
+```javascript
+d3.statosio( source, x, y, options )
+``````
```````
extra fences need removing

MattIPv4 commented on statosio/select-and-change-position-of-columns/index.md:
```diff
@@ -0,0 +1,223 @@
+
+## Result
+
+![# d3.statosio](https://d3.statosio.com/assets/images/example-select-400.jpg)<br>
```
```suggestion
![# d3.statosio](https://d3.statosio.com/assets/images/example-select-400.jpg)
```
br not needed

MattIPv4 commented on statosio/select-and-change-position-of-columns/index.md:
```````diff
@@ -0,0 +1,223 @@
+
+```javascript
+d3.statosio( source, x, y, options )
+``````
```````
Extra fences need to be removed

MattIPv4 commented on statosio/sort-dataset-by-values/index.md:
```diff
@@ -0,0 +1,224 @@
+
+## Result
+
+![# d3.statosio](https://d3.statosio.com/assets/images/example-sort-400.jpg)<br>
```
```suggestion
![# d3.statosio](https://d3.statosio.com/assets/images/example-sort-400.jpg)
```
br isnt needed

MattIPv4 commented on statosio/sort-dataset-by-values/index.md:
```````diff
@@ -0,0 +1,224 @@
+
+```javascript
+d3.statosio( source, x, y, options )
+``````
```````
Extra fences need removing

MattIPv4 reviewed (changes requested):
Tyvm :)

MattIPv4 reviewed (approved):
Sorry for the delay -- all lgtm, tyvm!
